### PR TITLE
Add paste icon inside invitation token entry

### PR DESCRIPTION
**Pull request by muralidharan1001**
fixes[#50](https://github.com/maidsafe/safe-authenticator-mobile/issues/50)

-added a box view under the entry and paste icon which will bring the icon inside invitation token entry.
-created a property in the material entry to make the underline transparent.
-created a new content view for the invitation entry along with the paste button in it as a grid.
-created a data trigger for changing the color of the box view when there is a focus in the entry.

### CODE REVIEW

**Comment by ravinderjangra** [comment]
hey @muralidharan1001, The implementation is correct. It'll be great if you can add screenshots (Android and iOS) also for these changes.

**Comment by muralidharan1001** [comment]
**screenshots of the invitation token entry for android**

-when there is a focus on the entry:

entryfocused_android

-when there is no focus on the entry:

entryunfocused_android


**screenshots of the invitation token entry for ios**

-when there is a focus on the entry:

entryfocused_ios

-when there is no focus on the entry:

entryunfocused_ios

File: SafeAuthenticator/Controls/InvitationTokenEntryView.xaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                     Grid.Row="1"
+                     HorizontalOptions="FillAndExpand" 
+                     Grid.Column="0"
+                     Grid.ColumnSpan="2" 
+                     HeightRequest="1.5">
+                <BoxView.Triggers>
+                    <DataTrigger Binding="{Binding Source={x:Reference InvitationTokenEntry}, Path=IsFocused}" 
+                                 Value="True"
+                                 TargetType="BoxView" >
+                        <Setter Property="BackgroundColor" Value="#0074e4"/>
```
Comment (ravinderjangra) [inline SafeAuthenticator/Controls/InvitationTokenEntryView.xaml]: Existing `StaticResources` can be used.

File: SafeAuthenticator/Controls/InvitationTokenEntryView.xaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                   Grid.Column="1"
+                   HorizontalOptions="End">
+                <Image.GestureRecognizers>
+                    <TapGestureRecognizer NumberOfTapsRequired="1"
+                                          Command="{Binding ClipboardPasteCommand}" />
+                </Image.GestureRecognizers>
+            </Image>
+
+            <BoxView x:Name="underline"
+                     BackgroundColor="#80000000"
```
Comment (ravinderjangra) [inline SafeAuthenticator/Controls/InvitationTokenEntryView.xaml]: Existing `StaticResources` can be used.

**Comment by muralidharan1001** [comment]
The requested changes are updated
- Added a static resource for the box view and it is used.
- The extra space in the Invitation token entry is reduced.

**Pull request merged**